openbb_platform/core/openbb_core/app/service/hub_service.py

```diff
@@ -1,27 +1,39 @@
 """Hub manager class."""
 
 from typing import Optional
+from warnings import warn
 
 from fastapi import HTTPException
 from jose import JWTError
 from jose.exceptions import ExpiredSignatureError
 from jose.jwt import decode, get_unverified_header
 from openbb_core.app.model.abstract.error import OpenBBError
 from openbb_core.app.model.credentials import Credentials
-from openbb_core.app.model.hub.features_keys import FeaturesKeys
 from openbb_core.app.model.hub.hub_session import HubSession
 from openbb_core.app.model.hub.hub_user_settings import HubUserSettings
 from openbb_core.app.model.profile import Profile
 from openbb_core.app.model.user_settings import UserSettings
 from openbb_core.env import Env
-from pydantic import SecretStr
 from requests import get, post, put
 
 
 class HubService:
     """Hub service class."""
 
     TIMEOUT = 10
+    # Mapping of V3 keys to V4 keys for backward compatibility
+    V3TOV4 = {
+        "API_KEY_ALPHAVANTAGE": "alpha_vantage_api_key",
+        "API_BIZTOC_TOKEN": "biztoc_api_key",
+        "API_FRED_KEY": "fred_api_key",
+        "API_KEY_FINANCIALMODELINGPREP": "fmp_api_key",
+        "API_INTRINIO_KEY": "intrinio_api_key",
+        "API_POLYGON_KEY": "polygon_api_key",
+        "API_KEY_QUANDL": "nasdaq_api_key",
+        "API_TRADIER_TOKEN": "tradier_api_key",
+        "API_ULTIMA_KEY": "ultima_api_key",
```

Comment:
@montezdesousa I believe we don't have ULTIMA on the OpenBB Platform anymore?

Reply:
will remove it tks